src/LibraryManager/Providers/BaseProvider.cs

```diff
@@ -190,9 +194,9 @@ protected async Task<ILibraryOperationResult> WriteToFilesAsync(ILibraryInstalla
                             return new LibraryOperationResult(state, PredefinedErrors.CouldNotWriteFile(file));
                         }
 
+                        string sourcePath = GetCachedFileLocalPath(state, file);
                         string destinationPath = Path.Combine(state.DestinationPath, file);
-                        var sourceStream = new Func<Stream>(() => GetStreamAsync(state, file, cancellationToken).Result);
```

Comment:
still used?

Reply:
You mean GetStreamAsync?  It was removed below.

Reply:
Ahh, missed that as I miss codeflow